### PR TITLE
Adding dashboard for MinIO node metrics

### PR DESCRIPTION
**Pull request by shtripat**
## Community Contribution License
All community contributions in this pull request are licensed to the project maintainers
under the terms of the [Apache 2 license](https://www.apache.org/licenses/LICENSE-2.0). 
By creating this pull request I represent that I have the right to license the 
contributions to the project maintainers under the Apache 2 license.

## Description
Added new grafana dashboard for visualizing the Node specific metrics

## How to test this PR?
Load the dashboard in grafana console against a data source referring `/minio/v2/metrics/node`

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Optimization (provides speedup with no functional changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Unit tests added/updated
- [ ] Internal documentation updated
- [ ] Create a documentation update request [here](https://github.com/minio/docs/issues/new?label=doc-change,title=Doc+Updated+Needed+For+PR+github.com%2fminio%2fminio%2fpull%2fNNNNN)

### CODE REVIEW

**Review by harshavardhana — APPROVED** [review]

File: docs/metrics/prometheus/grafana/node/minio-node.json [hunk 1]
```diff
@@ -0,0 +1,953 @@
...
+      },
+      "pluginVersion": "10.4.0",
+      "targets": [
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "${DS_PROMETHEUS}"
+          },
+          "exemplar": true,
+          "expr": "max(minio_node_drive_total{job=~\"$scrape_jobs\"})",
```
Comment (midorinet) [inline docs/metrics/prometheus/grafana/node/minio-node.json:116]: isn't it should be

```
sum(minio_node_drive_total{job=~\"$scrape_jobs\"})
```

File: docs/metrics/prometheus/grafana/node/minio-node.json [hunk 2]
```diff
@@ -0,0 +1,953 @@
...
+      "pluginVersion": "10.4.0",
+      "targets": [
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "${DS_PROMETHEUS}"
+          },
+          "editorMode": "code",
+          "exemplar": false,
+          "expr": "max(minio_node_drive_online_total{job=~\"$scrape_jobs\"})",
```
Comment (midorinet) [inline docs/metrics/prometheus/grafana/node/minio-node.json:184]: Isn't it should be

```
sum(minio_node_drive_online_total{job=~\"$scrape_jobs\"})
```

File: docs/metrics/prometheus/grafana/node/minio-node.json [hunk 3]
```diff
@@ -0,0 +1,953 @@
...
+          "step": 60
+        },
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "${DS_PROMETHEUS}"
+          },
+          "editorMode": "code",
+          "exemplar": false,
+          "expr": "max(minio_node_drive_offline_total{job=~\"$scrape_jobs\"})",
```
Comment (midorinet) [inline docs/metrics/prometheus/grafana/node/minio-node.json:203]: should be

```
sum(minio_node_drive_offline_total{job=~\"$scrape_jobs\"})
```

**Review by midorinet — COMMENTED** [review]
isn't it should be `sum` instead of `max` ?

**Comment by shtripat** [comment]
> isn't it should be `sum` instead of `max` ?

Ah, I feel there should not be max/sum at all. These are total count already.
Will send a PR to correct it.

**Pull request merged**